packages/client/src/runtime/core/jsonProtocol/serializeJsonQuery.test.ts

```diff
@@ -577,24 +577,29 @@ test('args - object with $type field', () => {
       args: { where: { jsonColumn: { $type: 'Decimal', value: '123' } } },
     }),
     // not using inline snapshot here because our default serializer mangles backslashes on windows
-  ).toBe(`{
-  "modelName": "User",
-  "action": "findMany",
-  "query": {
-    "arguments": {
-      "where": {
-        "jsonColumn": {
-          "$type": "Json",
-          "value": "{\\"$type\\":\\"Decimal\\",\\"value\\":\\"123\\"}"
+  ).toMatchInlineSnapshot(`
+    {
+      "modelName": "User",
+      "action": "findMany",
+      "query": {
+        "arguments": {
+          "where": {
+            "jsonColumn": {
+              "$type": "Raw",
+              "value": {
+                "$type": "Decimal",
+                "value": "123"
+              }
+            }
+          }
+        },
+        "selection": {
+          "$composites": true,
+          "$scalars": true
         }
       }
-    },
-    "selection": {
-      "$composites": true,
-      "$scalars": true
     }
-  }
-}`)
+  `)
```

Comment:
Suggestion: keep the old test too so making it
- `args - object with $type field (Json)`
- `args - object with $type field (Raw)`